eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/rest/ClusteredVariantsRestController.java

```diff
@@ -49,5 +63,30 @@ public ClusteredVariantsRestController(
             @PathVariable List<Long> identifiers) {
         return basicRestController.get(identifiers);
     }
+
+    @ApiOperation(value = "Find submitted variants by clustered variant identifier")
+    @GetMapping(value = "/{identifiers}/submitted", produces = "application/json")
+    public List<AccessionResponseDTO<SubmittedVariant, ISubmittedVariant, String, Long>> getSubmittedVariants(
+            @PathVariable List<Long> identifiers) {
```

Comment:
No hard opinions here, but I think it would be worse to return a map, and as it is now it's more powerful and consistent (with the other enpoints) than if it asked for a single rs.

Reply:
on the other hand, asking for a single rs would make the interface tighter, show better the purpose of the endpoint, and avoid load abuse